Tasks/AzureRmWebAppDeployment/MSDeployUtility.ts

```diff
@@ -4,9 +4,23 @@
 
 import Q = require('q');
 import tl = require('vsts-task-lib/task');
+var fs = require('fs');
 var regedit = require('regedit');
 var azureRmUtil = require('./AzureRMUtil.js');
 
+export function fileExists(path) {
```

Comment:
Ok - you can add an optional options arg to check.  Point is you won't be able to write L0 tests.  Which is another subject.  You need to write those